package.json

```diff
@@ -22,5 +23,10 @@
     "express": "^4.17.1",
     "mysql2": "^2.0.2",
     "sequelize": "^5.21.2"
+  },
+  "devDependencies": {
+    "eslint": "^6.8.0",
```

Comment:
I added a linter to your project because it had very inconsistent style. I applied the automatic formatter, which is why so many lines changed.